src/Libt8.jl

```diff
@@ -41,7 +41,7 @@ end
 const ptrdiff_t = Cptrdiff_t
 
 # Definitions used from MPI.jl
-using MPI: MPI, MPI_Datatype, MPI_Comm, MPI_File
+using MPI: MPI, MPI_Datatype, MPI_Comm, MPI_Group, MPI_File
```

Comment:
```suggestion
using MPI: MPI, MPI_Datatype, MPI_Comm, MPI_File
```
`MPI_Group` is not used, is it? Can you also please update this also in the [prologue.jl](https://github.com/DLR-AMR/T8code.jl/blob/1f69af45b03af783a51995b4b344344af2b32a0e/dev/prologue.jl#L35) so this doesn't get generated in the future? Sorry, I think I missed that in https://github.com/DLR-AMR/T8code.jl/pull/61.

Reply:
Done!